src/pages/LogInPage.vue

```diff
@@ -0,0 +1,136 @@
+<template>
+  <div class="logInPage">
+  <div class="logInBlock">
```

Comment:
Keep indentation. Nested elements should have indentations relatively to the parent element